src/OpenLoco/OpenLoco.cpp

```diff
@@ -699,6 +717,8 @@ namespace OpenLoco
                 Input::handleKeyboard();
                 Audio::updateSounds();
 
+                Network::update();
```

Comment:
Do we know where vanilla did the update?

Reply:
It was blocking. When one player did a game command, the packet would be sent and the code would block until an ack was returned.